src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedVariantContextUtils.java

```diff
@@ -290,4 +289,42 @@ public static ReadPileup getPileup(final Locatable loc, final Iterable<GATKRead>
 
         return new ReadPileup(loc, pile);
     }
+
+    /** This is lifted directly from htsjdk!  However, it is a private method there.
+     *
```

Comment:
did you open an issue to expose this method?  if so, could you link to the issue here?

Reply:
https://github.com/samtools/htsjdk/issues/999